keda/README.md

```diff
@@ -64,8 +64,12 @@ their default values.
 | `asciiArt` | bool | `true` | Capability to turn on/off ASCII art in Helm installation notes |
 | `certificates.autoGenerated` | bool | `true` | Enables the self generation for KEDA TLS certificates inside KEDA operator |
 | `certificates.certManager.caSecretName` | string | `"kedaorg-ca"` | Secret name where the CA is stored (generatedby cert-manager or user given) |
+| `certificates.certManager.duration` | string | `"8760h0m0s"` | Certificate duration |
 | `certificates.certManager.enabled` | bool | `false` | Enables Cert-manager for certificate management |
 | `certificates.certManager.generateCA` | bool | `true` | Generates a self-signed CA with Cert-manager. If generateCA is false, the secret with the CA has to be annotated with `cert-manager.io/allow-direct-injection: "true"` |
+| `certificates.certManager.generateIssuer` | bool | `true` | Generates an Issuer resource with Cert-manager |
+| `certificates.certManager.issuer` | object | `{}` | Reference to custom Issuer. If generateIssuer is false, the issuer reference below has to be provided |
```

Comment:
Wouldn't it be better to define all the fields of the `issuer` object? CC @JorTurFer

Reply:
lol, I was reviewing the PR and I didn't see this :/ 
https://github.com/kedacore/charts/pull/530#discussion_r1339163017